packages/core/protocol-config.json

```diff
@@ -130,6 +141,20 @@
       "network_registry_contract_address": "0xf0D4D48866C4F9665212B29A64c14A1f0FEDFD4c",
       "tags": ["etherscan"],
       "network_registry_proxy_contract_address": "0x15e068Ef1f76319b1848b1fcB3e49D68724AEE07"
+    },
+    "monte-rosa": {
```

Comment:
We definitely should use `monterosa` instead with the `-`. Could be cause of some unexpected issues.

Reply:
`monte_rosa` could work as well

Reply:
fixed in fec940019717ed5c4a493f655b9f3c5e7768596c